app/api/orders.py

```diff
@@ -142,7 +142,8 @@ def after_create_object(self, order, data, view_kwargs):
 
             order_url = make_frontend_url(path='/orders/{identifier}'.format(identifier=order.identifier))
             for organizer in order.event.organizers:
-                send_notif_ticket_purchase_organizer(organizer, order.invoice_number, order_url, order.event.name)
+                send_notif_ticket_purchase_organizer(organizer, order.invoice_number, order_url, order.event.name,
+                                                     order.id)
```

Comment:
was it really necessary to add a new line?

Reply:
Yes, it exceeds 120 characters when I add it on the same line.